src/index.html

```diff
@@ -12,22 +12,24 @@
   <title>Document</title>
   <link rel="stylesheet" href="./style.css">
 </head>
+
 <body>
-<div id="header"></div>
+  <div id="header"></div>
 
-<div class="important"></div>
-<div class="interesting"></div>
-<div class="surprising"></div>
+  <div class="important"></div>
```

Comment:
do not change index.html, also, do not change indentation to 4 spaces

Reply:
done